src/health/health.e2e-spec.ts

```diff
@@ -0,0 +1,60 @@
+import { Test, TestingModule } from '@nestjs/testing';
+import { INestApplication } from '@nestjs/common';
+import * as request from 'supertest';
+import { HealthModule } from './health.module';
+
+describe('Health Module (e2e)', () => {
+   let app: INestApplication;
+
+   beforeEach(async () => {
```

Comment:
test/app.e2e-spec.tsもなんだけどこここれでいいよ
```suggestion
   beforeAll(async () => {
```

Reply:
app.e2eはインスタンス立ちまくるから元からbefore Allにしてると思う。ちなみにhealth checkはapp.e2eにまとめ忘れてるからこのファイルは消えます